include/executor/engine/memory.ipp

```diff
@@ -194,7 +202,11 @@ Expect<void>
 Executor::runStoreLaneOp(Runtime::StackManager &StackMgr,
                          Runtime::Instance::MemoryInstance &MemInst,
                          const AST::Instruction &Instr) {
+#if defined(_MSC_VER) && !defined(__clang__) // MSVC
```

Comment:
This could be replaced with the above modifications.

Reply:
Fixed in https://github.com/WasmEdge/WasmEdge/pull/2751/commits/83d3d8e1c4cc1e2f5c67eb3d4aad6ba9cd778fc4#diff-7f1822f82a75e31952730d9cbe7d4891290088cd65ca9b19c5a8b5c03068a80c